### ISSUE TITLE
NUM_CPUS_PR_NODE vs FORWARD_MODEL specifying cpus

### ISSUE THREAD

**Issue by oysteoh** [equinor/ert#6054]
Noticed while tuning configuration on Azure runs that if you specify NUM_CPUS_PR_NODE = 1 and then you add a parameter to the eclipse forward model stating it can use 2 cpu's - ERT just dies with `core dumped` - no other error message. Not sure if we can do anything here?

**Comment by oysteoh** [comment]
NUM_CPUS_PR_NODE is to be removed in your milestone now @xjules and thus we can close this issue?

**Comment by xjules** [comment]
Yes, I think so. Will refer this in our milestone just to be sure :+1:

**Comment by oysteoh** [comment]
I'll pass this issue to your squad @xjules , feel free to close it if not relevant anymore 👍

**Comment by berland** [comment]
Fixed by #8163
